### PR TITLE
[Zone] Barrens

### PR DESCRIPTION
**Pull request by Knindzagxg**
Barrens is now fully updated, since every creature in zone was missing SmartAI.

Working on every zone at the moment, and will post updates once i'm done with each.

To test it:
Apply SQL
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID
or
.tele Barrens and run / test around.

### CODE REVIEW

**Comment by Geekena** [comment]
damage and spells are all ok
no problems found

File: data/sql/updates/pending_db_world/rev_1590935951123059600.sql [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+(3280, 0, 0, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 6728, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Witchwing Windcaller - Between 20-80% Health - Cast \'6728\' (No Repeat)');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 3278;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 3278);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(3278, 0, 0, 0, 0, 0, 100, 1, 4000, 6500, 0, 0, 11, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Witchwing Slayer - In Combat - Cast \'0\' (No Repeat)'),
```
[the remark this answers is not in the document]
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1590935951123059600.sql:221]: Damn, how the heck i missed this.
Updating.

[the remark this answers is not in the document]
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1590935951123059600.sql]: Data is mostly from different DB and online sources.
Rest, is taken from classic with addon that collect timer and spells which have been cast.

**Comment by merdlin** [comment]
Tested, it works.

**Pull request merged**